backtrace-library/src/main/cpp/include/crashpad-backend.h

```diff
@@ -0,0 +1,31 @@
+#ifndef BACKTRACE_ANDROID_CRASHPAD_BACKEND_H
+#define BACKTRACE_ANDROID_CRASHPAD_BACKEND_H
+
+#include "base/logging.h"
+#include "client/crashpad_client.h"
+#include "client/crashpad_info.h"
+#include "client/crash_report_database.h"
+#include "client/settings.h"
+
+#include "client-side-unwinding.h"
+
+#include <jni.h>
+
+// crashpad client
+using namespace base;
```

Comment:
We should strive to avoid using `using namespace`, and never do this in the global namespace in a header.

If we _must_ use `using namespace`, at the very most, do it at a function/class/inner namespace level.

Reply:
Sounds good, will just fully qualify these symbols